cfn/templates/ecs-task.template.yml

```diff
@@ -0,0 +1,164 @@
+        {%- if environment is mapping %}
+        {# blankline #}
+          Environment:
+          {%- for k, v in environment.items() %}
+            {# blankline #}
+            - Name: {{ k }}
+            {#-
+                Deal with Map/List data Types
+            #}
+            {%- if v is mapping %}
+            {#- This feels brittle, https://arcpublishing.atlassian.net/browse/ST-1240 #}
+            {%- set vl = v.keys() | list %}
+            {%- if vl is iterable and vl|length > 0 and ("Fn::" == vl[0][0:4] or "Ref" == vl[0]) %}
+              {#- Map is a translated !Fn object #}
+              Value: {{ v | tojson }}
+            {%- else %}
+              {#- Convert from map to json string, wrap in singleQuotes #}
+              Value: '{{ v | tojson }}'
+            {%- endif %}
+            {%- elif v is sequence and v is not string %}
+              {#- Convert from list to json string, wrap in singleQuotes #}
+              Value: '{{ v | tojson }}'
+            {#-
+                Deal with non-string types
+            #}
+            {%- elif v is number or v is sameas true or v is sameas false %}
+              {#- Item is Number or Boolean #}
+              Value: '{{ v }}'
+            {%- elif v is not string %}
+              {#- Unknown datatype, to not tamper #}
+              Value: {{ v }}
+            {#-
+                Process AWS CFN Specials
+            #}
+            {%- elif v[0] == '{' and 'resolve:' in v %}
+              {#- Inline CFN Dynamic Reference, wrap in single quotes #}
+              {#- https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/dynamic-references.html #}
+              Value: '{{ v }}'
+            {%- elif v[0] in '[{!' %}
+              {#- No Quotes for values which look like CFN/JSON #}
+              Value: {{ v }}
+            {%- elif '${AWS:' in v %}
+              {#- if ${AWS: in string then add CFN !Sub to allow inline substitutions #}
+              Value: !Sub '{{ v }}'
+            {#-
+                Process items with quotes
+            #}
+            {%- elif '"' in v or "'" in v %}
+              {#- No quotes for objects containing quotes #}
+              Value: {{ v }}
+            {#-
+                All others are singleQuoted
+            #}
+            {%- else %}
+              {#- Otherwise wrap in single quotes #}
+              Value: '{{ v }}'
+            {%- endif %}
+          {%- endfor %}
+        {%- endif %}
+      
+          
+  KlaxonTaskSchedule:
+    Type: AWS::Events::Rule
+    Properties:
+      # Description: String
+      # EventBusName: String # optional
+      # EventPattern: Json # optional
+      # Name: String
+      # RoleArn: String
+      # ScheduleExpression: String
+      # State: String
+      # Targets: 
+      #   - Target
+      Description: Klaxon ecs task scheduler
+      Name: klaxon-ecs-task-scheduler
+      RoleArn: arn:aws:iam::912288704264:role/ECSTaskExecutionRole
```

Comment:
From the documentation: 
> RoleArn
> 
> The Amazon Resource Name (ARN) of the role that is used for target invocation.
> 
> If you're setting an event bus in another account as the target and that account granted permission to your account through an organization instead of directly by the account ID, you must specify a RoleArn with proper permissions in the Target structure, instead of here in this parameter. 
 
 
 Using the task execution role  for this purpose seeeems like the right move based on this documentation. Could we reference this another way like `!GetAtt ECSTaskExecutionRole.Arn`?